src/validations/class.js

```diff
@@ -2,8 +2,8 @@ const Joi = require('joi');
 
 const validateCreation = (req, res, next) => {
   const classValidation = Joi.object({
-    activity: Joi.string().min(3).required(),
-    trainer: Joi.string().min(3).required(),
+    activity: Joi.string().hex().length(24).required(),
+    trainer: Joi.array().items(Joi.string().hex().length(24)).required(),
```

Comment:
Even though the changes seem syntactically correct and the server is able to run, there being no data uploaded to the MongoDB database, there's no real way of knowing if it works as intended. Upload class.json to the database so that I can test the endpoints.

Reply:
I apologize for the oversight. Thank you for bringing it to my attention. The issue has been resolved, and the corrected code has been implemented. I have now uploaded the file to the MongoDB database. You can proceed with testing the endpoints as intended. Please feel free to reach out if you have any further questions or concerns.